packages/composite-checkout/src/components/wpcom/items.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+import { renderDisplayValueMarkdown } from '../../lib/render';
+
+export function PlanItem( { plan, onDeleteItem, onChangePlanLength } ) {
+	const changePlanLength = planLength => onChangePlanLength( plan, planLength );
+	const deleteItem = () => onDeleteItem( plan );
+	return (
+		<React.Fragment>
+			<div>
+				<span>
+					<div>{ plan.label }</div>
+					{ plan.subLabel && <div>{ plan.subLabel }</div> }
+				</span>
+				<span>{ renderDisplayValueMarkdown( plan.amount.displayValue ) }</span>
+				<button onClick={ deleteItem } />
```

Comment:
If you add a string child to the `button` element, it will appear as the button text.